Package.swift

```diff
@@ -165,6 +165,14 @@ extension Target {
     ]
   )
 
+  static let swiftGRPCPlugin: Target = .plugin(
+    name: "SwiftGRPCPlugin",
```

Comment:
This should be "GRPCSwiftPlugin"

Reply:
I agree but it is kinda sad that this isn't aligned with the `SwiftProotobufPlugin` then

Reply:
I don't mind changing it to `GRPCSwiftPlugin` (it makes more sense since the repo's called grpc-swift), but as Franz said, the reason why I went for SwiftGRPC was to keep it consistent with the `SwiftProtobufPlugin`. If we don't care about that I can change it.